init.ps1

```diff
@@ -13,7 +13,11 @@ param(
     [boolean]$addOctopack = $true,
     [boolean]$addUnitTests = $true,
     [boolean]$addRebuildDatabase = $true,
-    [string]$branch = "master"
+    [string]$branch = "master",
+	[boolean]$importTemplateRepo = $true,

```

Comment:
Yeah I originally did make it default to false.  This makes more sense but it was easier to test to have it default to true  I keep forgetting to select true and had to re run the whole process.